.github/workflows/release.yml

```diff
@@ -42,3 +48,7 @@ jobs:
       uses: rajatjindal/krew-release-bot@v0.0.38
       with:
         krew_template_file: .krew/ns.yaml
+    - name: Publish Snaps to the Snap Store (stable channel)
+      run: for snap in $(ls dist/*.snap); do snapcraft upload --release=stable $snap; done
+      env:
+        SNAPCRAFT_STORE_CREDENTIALS: ${{ secrets.STORE_LOGIN }}
```

Comment:
I just realized I added a `$SNAPCRAFT_TOKEN` to the repo last year. Can we just use that?

Reply:
Done